application.go

```diff
@@ -1,35 +1,31 @@
-package client
+package sdk
 
 import (
 	"context"
 
 	"github.com/cosmos/gogoproto/grpc"
 	"github.com/pokt-network/poktroll/x/application/types"
-
-	"github.com/pokt-network/shannon-sdk/sdk"
 )
 
-var _ sdk.ApplicationClient = (*applicationClient)(nil)
```

Comment:
Why is `Application` no longer complying with `sdk.ApplicationClient` interface?

Reply:
It is part of moving interfaces to consumer side: the producer should accompany breaking changes to any public functions/methods with proper semantic versioning, rather than referencing the interface on the producer side.